plugins/pdal/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 3.21)
+
+project(f3d-plugin-pdal)
+
+include(GNUInstallDirs)
+
+# Check if the plugin is built externally
+if(PROJECT_IS_TOP_LEVEL)
+  find_package(f3d REQUIRED COMPONENTS pluginsdk)
+else()
+  include(f3dPlugin)
+endif()
+
+f3d_plugin_init()
+
+f3d_plugin_declare_reader(
+    NAME PDAL
+    EXTENSIONS arrow bpf ilvis2 e57 fbi mbio npy csd pcd ply qfit rdb rxp sbet bin tiledb las drc ept.json json hdf h5 mat ntif obj pts ptx txt splk spz
```

Comment:
We do not want to add all these extensions, lets add:

1. The one we actually want : e57 gdal las 
2. The one we can test: I leave it up to you. but each of the individual format will need a test.
3. And remove the one that we dont want to use PDAL for: obj, pts, spz, ply, npy,